#! /bin/sh /usr/share/dpatch/dpatch-run
## libgpsmm.h-fix.dpatch by Bernd Zeimetz <bzed@debian.org>
##
## DP: no need to include gpsd_config.h here.

@DPATCH@
diff -urNad gpsd~/libgpsmm.h gpsd/libgpsmm.h
--- gpsd~/libgpsmm.h	2006-12-15 00:43:39.000000000 +0100
+++ gpsd/libgpsmm.h	2007-12-05 23:01:37.000000000 +0100
@@ -10,7 +10,6 @@
 #define _GPSMM_H_
 
 #include <sys/types.h>
-#include "gpsd_config.h"
 #include "gps.h" //the C library we are going to wrap
 
 class gpsmm {
